python/graphstorm/inference/utils.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utility functions for inference
+"""
+
+import os
+import shutil
+import tarfile
+
+
+def wrap_model_artifacts(path_to_model, path_to_yaml, path_to_json, path_to_entry,
+                         wrap_name='model', output_path=None):
+    """ A utility function to zip model artifacts into a tar package
+
+    This function will put the entry point file into a sub-folder,named `code`, and then zip the
+    `model.bin`, the `**.yaml`, the `**.json` file, and the `code` sub-folder into a tar package
+    with the name specified by the wrap_name argument, and save it to the `output_path`.
+
+    This function will check if given files exist. Non-existance will trigger an error raising.
+    
+    Parameters
+    ----------
+    path_to_model: str
+        The path of GraphStorm's `model.bin` file.
```

Comment:
What about learnable embeddings?

Reply:
For realtime inference, we don't support using the learnable embeddings. So, in the current implementation, not wrap the learnable embeddings.